test/unit/registry-domain-plugins-initialiser.js

```diff
@@ -105,29 +105,24 @@ describe('registry : domain : plugins-initialiser', () => {
         {
           name: 'getValue',
           register: {
-            register: function(options, deps, cb) {
+            register: (options, deps, cb) => {
               passedOptions = options;
               cb();
             },
-            execute: function(key) {
-              return passedOptions[key];
-            }
+            execute: key => passedOptions[key]
           },
-          options: {
-            a: 123,
-            b: 456
-          }
+          options: { a: 123, b: 456 }
         },
         {
           name: 'isFlagged',
           register: {
-            register: function(options, deps, cb) {
-              flag = true;
-              cb();
+            register: (options, deps, cb) => {
+              setTimeout(() => {
+                flag = true;
+                cb();
+              }, 10);
```

Comment:
With this artificial timeout, I am testing that the callback is actually called after the registration (test failed before the code change)